regression/cbmc/struct15/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--trace --smt2
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
```

Comment:
:pick: Might be nice to have a regex in the must not be matched section here checking for `map::at`.

Reply:
Good idea, thanks! Pushed this change.